mh/quest_utils.py

```diff
@@ -0,0 +1,1564 @@
+import struct
+
+
+LAYER_CHAT_COLORS = (0xbb3385ff, 0xffffffff, 0xffffffff)
+
+QUEST_FLAG_UNK = 0b1000  # It really is unknown, it put a question mark icon on the monster/item box icon
+QUEST_FLAG_ARENA = 0b10000
+QUEST_FLAG_DOUBLE_MONSTER = 0b10000
+
+
+def pad(s, size, p=b'\0'):
+    data = bytearray(s + p * max(0, size-len(s)))
+    data[-1] = 0
+    return data
```

Comment:
This function already exists in `mh/constants.py`, if you really need it you should move the one from the `mh.constants` module into a more relevant place like `other/utils.py` and import it rather than copy-paste it.